src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/iot_stack.py

```diff
@@ -165,10 +169,14 @@ def setup_custom_authorizer(self):
             "custom_authorizer_token_signature"
         ] = custom_authorizer_token_signature
 
-        authorizer_function_arn = self.setup_custom_authorizer_function()
+        authorizer_function_arn = self.setup_custom_authorizer_function("1",
+                                                                        "custom_resources/iot_custom_authorizer_function",
```

Comment:
Indent style chomp down or wrap & indent? This is very far to the right of the screen. I'd do +4 or +8 off the previous line, instead.

Reply:
went with +8 on the next line, thanks!

Reply:
`flake8` and `black` should catch this (and autofix some of it)